### ISSUE TITLE
The 'add' command fails bizarrely if a path is passed

### ISSUE THREAD

**Issue by nventuro** [OpenZeppelin/openzeppelin-sdk#1217]
`oz add` expects a contract name, but if a path is passed instead (which is both natural and easy due to shell filename autocomplete), the command will fail with an unrelated error which originates from the built artifacts being stored in a single flat directory.

```bash
$ npx oz add contracts/MyContract.sol 
Nothing to compile, all contracts are up to date.
Contract contracts/MyContract.sol not found in path ~/project/build/contracts/contracts/MyContract.sol.json
```

Even if we choose not to accept paths in `add`, the error message should be better.

**Comment by jcarpanelli** [comment]
Indeed, the message error is not the best :sweat_smile:. I found this suggestion as a good DX improvement, but right now we are aiming to kill the `oz add-push-create` command sequence in favor of just running `oz create`, which runs the `add` and `push` commands under the hood. Hopefully, we'll remove all the unnecessary commands soon!

**Comment by nventuro** [comment]
To be fair, I was doing `add` because I was under the impression it was required in order to compile a contract. If deploying is the only reason to add a contract, moving everything to `create` makes a lot of sense.

**Comment by abcoathup** [comment]
Just need to do `npx oz create`
